kratos/utilities/sensitivity_builder.cpp

```diff
@@ -217,9 +217,12 @@ void AssembleNodalSolutionStepContainerContributions(const SensitivityVariables<
             return;
         builder.CalculateLocalSensitivity(*rVariables.pDesignVariable, rElement,
                                           rResponseFunction, rProcessInfo);
+        // if rElement does not contribute to local sensitivity, skip assembly
+        if (builder.LocalSensitivity.size() != 0) {
```

Comment:
can you fix the indention in the next lines?

Reply:
@armingeiser I fixed it.